pokepro/src/Pages/Pokemon/Pokemon.tsx

```diff
@@ -47,8 +54,11 @@ function Pokemon() {
                 </div>
               </div>
               <div className="w-1/2 h-full text-red-400 flex flex-col pl-12 pt-6 justify-between">
-                {currentPokemon.type.map((item) => (
-                  <div className=" w-fit bg-green-200 capitalize rounded-md pl-4 pr-4  ">
+                {currentPokemon.type.map((item, idx) => (
+                  <div
+                    key={idx}
```

Comment:
Using index as a key is not recommend, lets use a id from data or name from the data we have. because it has to be unique

```suggestion
                {currentPokemon.type.map((item) => (
                  <div
                    key={`pokemon-${type}`}
```